.licensed.yml

```
@@ -13,4 +13,10 @@ allowed:
 
 reviewed:
   npm:
-  - sax
\ No newline at end of file
+  - sax
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 4 but found 2

(indentation)